state/executor.go

```diff
@@ -169,7 +169,7 @@ func (e *BlockExecutor) CreateBlock(height uint64, lastCommit *types.Commit, las
 	}
 
 	block.Data.Txs = toRollkitTxs(txl)
-	block.SignedHeader.LastCommitHash = lastCommit.GetCommitHash(&block.SignedHeader.Header, e.proposerAddress)
+	block.SignedHeader.LastCommitHash = lastSignature.GetCommitHash(&block.SignedHeader.Header, e.proposerAddress)
```

Comment:
`LastCommitHash` needs at least comment in the code, that it's the hash of ABCI Commit, because it's getting confusing with `lastSignature` IMHO.

Reply:
Added